apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -103,11 +122,25 @@ class InlineAudio extends React.Component {
     playing: false,
     error: false,
     hover: false,
-    loaded: false
+    loaded: false,
+    autoplayed: false
   };
 
+  constructor(props) {
+    super(props);
+    this.autoplayAudio = this.autoplayAudio.bind(this);
+  }
+
   componentDidMount() {
     this.getAudioElement();
+    if (this.props.ttsAutoplayEnabled && !this.state.autoplayed) {
+      const {autoplayTriggerElementId} = this.props;
+      this.autoplayTriggerElement = autoplayTriggerElementId
```

Comment:
NIT: i'm personally of the opinion that we should declare all instance properties in or under the constructor (depending on whether or not they need access to `this`) even when they start out as null as a sort of "directory" for future devs, so i'd recommend adding `autoplayTriggerElement = null` around line 33